2024-08-26 02:53:49,131 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 7203280ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 04:54:52,806 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 7203845ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 06:57:06,088 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 7213010ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 09:00:42,635 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 7216556ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 13:20:21,254 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 1816619ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 14:01:35,031 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 2414135ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 18:36:17,116 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 1799898ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 19:06:32,756 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 1815604ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 19:22:36,688 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 903974ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 19:40:07,672 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 990951ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 20:00:31,570 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 1163954ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 20:38:22,482 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 2150833ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 21:14:35,395 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 2112881ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 21:32:07,276 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 991922ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-26 22:57:49,219 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 5081960ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-27 00:38:15,116 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 5966020ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

